#!/usr/bin/python3
def raise_exception_msg(message=''):
    ne = message
    raise NameError
